### PR TITLE
feat: pod create functions

### PR DESCRIPTION
**Pull request by 0xGrapefruit**

### CODE REVIEW

**Review by DanThomp507 — APPROVED** [review]

File: src/pod-create.ts [hunk 1]
```diff
@@ -1,13 +1,190 @@
...
+  if (signer && !owners.includes(await signer.getAddress()))
+    throw new Error('Sender was not safe owner');
+
+  if (!(await isControllerEnabled(modules))) throw new Error('Pod module was not enabled');
+  // Checksum all addresses
+  const admin = args.admin ? ethers.utils.getAddress(args.admin) : ethers.constants.AddressZero;
+  try {
+    const MemberToken = getContract('MemberToken', config.provider);
+    const expectedPodId = (await MemberToken.getNextAvailablePodId()).toString();
+    const Controller = getContract('ControllerLatest', signer);
```
Comment (jtlacci) [inline src/pod-create.ts:165]: we can't rely on them having enabled latest module, we will probably need to figure out which module they have enabled

**Review by jtlacci — CHANGES_REQUESTED** [review]

**Review by jtlacci — COMMENTED** [review]
are we adding a function to create the pod image

File: src/config.ts [hunk 2]
```diff
@@ -10,7 +10,8 @@ export const config = {
   subgraphUrl: null,
   gnosisUrl: null,
   etherscanUrl: null,
-  etherscanApiKey: null,
+  etherscanApiKey: '2WNX6IRB7FI49FW9BPY2W8YE5XGBCDQCRB',
```
Comment (jtlacci) [inline src/config.ts:13]: we shouldn't be commiting this

**Review by jtlacci — APPROVED** [review]
Looks great will

**Pull request merged**